resources/js/components/ShowReminderModal.vue

```diff
@@ -5,32 +5,47 @@
+                        <p class="text-grey-dark text-xl mb-4">People</p>
+                        <div class="flex mb-4" v-if="reminder.guests.length !== 0">
+                            <div class="flex bg-grey-lighter rounded-full w-36 items-center p-1 mr-2" v-for="guest in reminder.guests" :key="guest.id">
+                                <img class="h-10 w-10" :src="guest.avatar_path" alt="Avatar">
+                                <span class="truncate" v-text="guest.username"></span>
+                            </div>
                         </div>
                     </div>
-                </div>
-                <div class="flex flex-col mb-6" v-if="reminder.description">
-                    <span class="text-grey-dark text-xl mb-2">Description</span>
-                    <span class="bg-grey-lighter rounded-xl p-2">{{ reminder.description }}</span>
-                </div>
-                <div class="flex justify-end pb-6 mt-2">
-                    <div class="flex items-center" @click="window.alert('edit')">
-                        <img class="cursor-pointer h-4 mr-1" src="/img/delete.svg"/>
-                        <a class="text-sm font-semibold text-grey-dark underline mr-8 cursor-pointer">DELETE</a>
+                    <div class="flex flex-col mb-6" v-if="reminder.description">
+                        <span class="text-grey-dark text-xl mb-2">Description</span>
+                        <span class="bg-grey-lighter rounded-xl p-2">{{ reminder.description }}</span>
                     </div>
-                    <div class="flex items-center" @click="window.alert('edit')">
-                        <img class="cursor-pointer h-4 mr-1" src="/img/pencil.svg"/>
-                        <a class="text-sm font-semibold text-grey-dark underline cursor-pointer">EDIT</a>
+                    <div class="flex justify-end pb-6 mt-2">
+                        <div class="flex items-center" @click="showDeletePrompt = true">
+                            <img class="cursor-pointer h-4 mr-1" src="/img/delete.svg"/>
+                            <a class="text-sm font-semibold text-grey-dark underline mr-8 cursor-pointer">DELETE</a>
+                        </div>
+                        <div class="flex items-center" @click="window.alert('edit')">
+                            <img class="cursor-pointer h-4 mr-1" src="/img/pencil.svg"/>
+                            <a class="text-sm font-semibold text-grey-dark underline cursor-pointer">EDIT</a>
+                        </div>
+                    </div>
+                </div>
+                <div v-else class=" pb-6 mt-2">
+                    <p class="text-4xl text-black font-bold mb-4">Are you sure you want delete the reminder?</p>
+                    <p class="text-xl text-grey">
+                        The reminder to be deleted is
+                        <span class="font-semibold">{{ reminder.title }}</span>.
+                    </p>
```

Comment:
Is this necessary though?

Reply:
I guess we'll have to arrange a UI/UX review at some point in the development where we'll tune it up.